### PR TITLE
Tweak error messages for the Save Branch as Scene editor option

### PR DESCRIPTION
**Pull request by Calinou**
See https://github.com/godotengine/godot-proposals/issues/2433.

### CODE REVIEW

File: editor/scene_tree_dock.cpp [hunk 1]
```diff
@@ -907,29 +907,32 @@ void SceneTreeDock::_tool_selected(int p_tool, bool p_confirm_override) {
...
 				accept->popup_centered();
 				break;
 			}
 
 			Node *tocopy = selection.front()->get();
 
 			if (tocopy == scene) {
-				accept->set_text(TTR("Can not perform with the root node."));
+				accept->set_text(
+						TTR("Can't save the root node branch as an instanced scene.\nTo create an editable copy of the current scene, duplicate it using the FileSystem dock context menu instead."));
```
Comment (KoBeWi) [inline editor/scene_tree_dock.cpp:928]: You could probably mention inheritance here too.

File: editor/scene_tree_dock.cpp [hunk 2]
```diff
@@ -907,29 +907,32 @@ void SceneTreeDock::_tool_selected(int p_tool, bool p_confirm_override) {
...
 				accept->popup_centered();
 				break;
 			}
 
 			List<Node *> selection = editor_selection->get_selected_node_list();
 
 			if (selection.size() != 1) {
-				accept->set_text(TTR("This operation requires a single selected node."));
+				accept->set_text(
+						vformat(TTR("Saving the branch as a scene requires selecting only one node, but you have selected %d nodes."), selection.size()));
```
Comment (akien-mga) [inline editor/scene_tree_dock.cpp:919]: I'm still not convinced by this kind of line break. We're splitting a very long line into one very short line + one very long line (one horizontal char saved with an 8-space tab like on GitHub).

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.4.

**Pull request merged**